in_toto/in_toto_run.py

```diff
@@ -142,7 +147,13 @@ def main():
     log.error("in load key - {}".format(args.key))
     sys.exit(1)
 
-  in_toto_run(args.step_name, args.materials, args.products,
+  if args.no_command:
+    in_toto_run(args.step_name, {}, {}, [], key, args.record_byproducts)
```

Comment:
We still want to be able to record the files someone is looking at (signing off). Please pass `args.materials` and `args.products` as 2nd and 3rd argument respectively.